store/server/features/employees/employeeManagment/mutations.ts

```
@@ -26,6 +26,19 @@ const createEmployee = async (values: any) => {
   });
 };
 
+
+const downloadEmployeeInfomation = async ({ downloadFormat, searchParams }: { downloadFormat: string, searchParams: Record<string, string> }) => {
```

ESLint

[error] Replace `⏎const·downloadEmployeeInfomation·=·async·({·downloadFormat,·searchParams·}:·{·downloadFormat:·string,·searchParams:·Record<string,·string>·` with `const·downloadEmployeeInfomation·=·async·({⏎··downloadFormat,⏎··searchParams,⏎}:·{⏎··downloadFormat:·string;⏎··searchParams:·Record<string,·string>;⏎`

(prettier/prettier)